plat/xilinx/zynqmp/pm_service/pm_common.h

```diff
@@ -21,13 +21,13 @@
 
 /**
  * pm_ipi - struct for capturing IPI-channel specific info
- * @mask	mask for enabling/disabling and triggering the IPI
- * @base	base address for IPI
+ * @apu_ipi_id	APU IPI agent ID
+ * @pmu_ipi_id	PMU Agent ID
  * @buffer_base	base address for payload buffer
  */
 struct pm_ipi {
-	const unsigned int mask;
-	const uintptr_t base;
+	const uint8_t apu_ipi_id;
+	const uint8_t pmu_ipi_id;
```

Comment:
The IDs are defined as 32-bit values and also passed around in the ipi_mb_XX_XX functions as uint32_t.